### ISSUE TITLE
Added a new collection (project) and having problems with searching

### ISSUE THREAD

**Issue by espenlg** [Stillat/documentation-search#1]
When I added a new project called "sjekkpunkt" and edited the search.php file to say `docs:sjekkpunkt` instead of `docs:documentation`. When I run `php please search:update` it says 0 entries even though there are 21 entries. So when trying to search I get an error saying "Empty stack", and this is probably because storage/statamic/search/docs.json is empty.

I tried to follow the documentation on documentation-search from a-z, but I'm at a loss 🤷‍♂️ Is there anything I can provide of information to help me figure this out?

**Comment by JohnathonKoster** [comment]
Hey there!

Do you have a template within `resources/views/documentation-search`? The easiest way to use them is to add a `_default.antlers.html` (or `_default.blade.php`) template at `resources/views/documentation-search/_default.antlers.html`.

That is the one way I know to get the behavior described

**Comment by espenlg** [comment]
Yes, that is in place, and I also tried renaming the "documentation"-folder to "sjekkpunkt". I have a feeling the indexing is the problem as the docs.json file is empty 🤔
